### PR TITLE
ToDo homework

### PR DESCRIPTION
**Pull request by angrybrain**

### CODE REVIEW

**Comment by angrybrain** [comment]
task 4(50%)

File: html-task-3/blocks/menu/menu.css [hunk 1]
```diff
@@ -1,68 +1,67 @@
...
+.menu--box {
     display: flex;
     width: 100%;
     justify-content: space-evenly;
     align-items: center;
     font-size: 15px;
 }
 
-.menu-box a:link {
+.menu--box a:link {
```
Comment (ClainAtomi) [inline html-task-3/blocks/menu/menu.css:16]: bem does not allow such selectors, ceparate class to `a`

File: html-task-3/blocks/menu/menu.css [hunk 2]
```diff
@@ -1,68 +1,67 @@
...
     color: gray;
     text-decoration: none;
     font-size: 14px;
     font-weight: 400;
     justify-content: center;
 }
 
-.menu-box a:hover {
+.menu--box a:hover {
```
Comment (ClainAtomi) [inline html-task-3/blocks/menu/menu.css:24]: bem does not allow such selectors, ceparate class to a

File: html-task-3/blocks/menu/menu.css [hunk 3]
```diff
@@ -1,68 +1,67 @@
...
     color: gold;
 }
 
-.menu-box a: {
+.menu--box a: {
```
Comment (ClainAtomi) [inline html-task-3/blocks/menu/menu.css:28]: why `:` here? :)

File: html-task-3/script.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    }
+}
+
+function ShowHTMLContent() {
+    content = this.innerHTML;
+    window.alert(content);
+}
+
+function highlightHeaderElements() {
+    background = document.getElementsByClassName("header--back")
```
Comment (ClainAtomi) [inline html-task-3/script.js:20]: good job here :)

File: js-task-4/script.js [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+    world.racePopulation = +(document.getElementById("configPopulation").value);
+    world.virusVirality = +(document.getElementById("configVirality").value);
+    world.virusDeadrate = +(document.getElementById("configDeadrate").value);
+    world.vactinationRate = +(document.getElementById("configVactinality").value);
+    world.cycle = +(document.getElementById("configSpeed").value);
+    world.vactineSpeed = world.cycle * 100;
+    world.virusSpeed = world.cycle * 100;
+    init();
+    // console.log(world)
+}
```
Comment (ClainAtomi) [inline js-task-4/script.js:212]: split entities into separate files :)

File: html-task-3/blocks/menu/menu.css (hunk 3 above)
Comment (angrybrain) [reply html-task-3/blocks/menu/menu.css]: copy\paste typo

File: html-task-3/blocks/menu/menu.css (hunk 2 above)
Comment (angrybrain) [reply html-task-3/blocks/menu/menu.css]: fixed

File: html-task-3/blocks/menu/menu.css (hunk 1 above)
Comment (angrybrain) [reply html-task-3/blocks/menu/menu.css]: fixed

**Pull request merged**